### PR TITLE
repart: Set sector size using SectorSize=

### PR DESCRIPTION
**Pull request by mcassaniti**

### CODE REVIEW

**Review by DaanDeMeyer — CHANGES_REQUESTED** [review]
Please modify the summary as well

File: mkosi/config.py [hunk 1]
```diff
@@ -996,6 +997,14 @@ class MkosiConfigParser:
             paths=("mkosi.repart",),
             help="Directory containing systemd-repart partition definitions",
         ),
+        MkosiConfigSetting(
+            dest="sector_size",
+            long="--sector-size",
+            name="SectorSize",
+            section="Output",
+            parse=config_parse_string,
+            help="Override the default sector size for disk images",
```
Comment (DaanDeMeyer) [inline mkosi/config.py:1006]: "Set the disk image sector size"

File: mkosi/config.py [hunk 2]
```diff
@@ -996,6 +997,14 @@ class MkosiConfigParser:
             paths=("mkosi.repart",),
             help="Directory containing systemd-repart partition definitions",
         ),
+        MkosiConfigSetting(
+            dest="sector_size",
+            long="--sector-size",
+            name="SectorSize",
```
Comment (DaanDeMeyer) [inline mkosi/config.py:1003]: This can be dropped, it's inferred automatically

**Review by DaanDeMeyer — CHANGES_REQUESTED** [review]

File: mkosi/config.py (hunk 2 above)
Comment (mcassaniti) [reply mkosi/config.py:1003]: Just drop long and name or all 3?

Comment (DaanDeMeyer) [reply mkosi/config.py:1003]: Drop long and name

**Review by DaanDeMeyer — APPROVED** [review]

**Pull request merged**